src/views/Home.jsx

```diff
@@ -90,33 +93,58 @@ export function Home({ setListToken }) {
 	};
 
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
+		<div className="home">
+			<p>Welcome to</p>
+			<h1>App Title!</h1>
+			<div className="app-logo">
+				<img src={appImage} alt="app logo" />
```

Comment:
Not sure if it would be helpful here, but there is also a bootstrap Image component.

Reply:
@sarahmjenkins Oh nice, I'll take a look at it.